Jax the Dragon

esemv07 opened the pull request.

acornitum commented:
Hey - is this finished? I noticed your code was cut off!

esemv07 commented:
Thanks for pointing that out, I didn't even notice. Just recommitted with the full code! Not sure why it was cut off.

acornitum commented:
This is great! I really like how cute that bearded dragon is. This is really polished! 

Small suggestion - when the pet is close to low health, could it play something other than the default animation? Maybe it's slower, maybe it looks really sad and is slouched down. 

This PR is really close to getting merged! I think it just needs a little more to be different from all the other "falling objects" games that I've gotten. 

_Note: don't open a new PR, just commit changes to this current thread + send a comment when changes have been made!_

esemv07 commented:
Thanks for the suggestions! Changes have now been made! I added a walking animation, and made it so that when the player's health is below 40, Jax is sad and starts sweating. I also added a start screen when the game is first run. Hope this is enough!

acornitum commented:
Thank you! This looks great - Jax is so cute.

I'm going to merge this! Fill out the form here: https://forms.hackclub.com/t/isDdJtrnrvus

Thanks so much for submitting!
